sdk/src/main/java/com/hedera/hashgraph/sdk/PrivateKey.java

```diff
@@ -75,6 +72,27 @@ public static PrivateKey generateECDSA() {
     }
 
     /**
+     * Extract the ED25519 private key from a seed.
+     *
+     * @param seed  the seed
+     * @return      the ED25519 private key
+     */
+    public static PrivateKey fromSeedED25519(byte[] seed) {
+        return PrivateKeyED25519.fromSeed(seed);
+    }
+
+    /**
+     * Extract the ECDSA private key from a seed.
+     *
+     * @param seed  the seed
+     * @return      the ECDSA private key
+     */
+    public static PrivateKey fromSeedECDSAsecp256k1(byte[] seed) {
+        return PrivateKeyECDSA.fromSeed(seed);
+    }
+
+    /**
+     * @deprecated use {@link Mnemonic#toStandardEd25519PrivateKey(String, int)} ()} or {@link Mnemonic#toStandardECDSAsecp256k1PrivateKey(String, int)} (String, int)} instead
```

Comment:
I think there needs to be a description somewhere about why this was deprecated. At least, the doc should mention that this function uses the incomplete derivation path